flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/nodes/CommonPythonCalc.scala

```diff
@@ -49,11 +48,10 @@ trait CommonPythonCalc {
   private[flink] def createPythonScalarFunctionInfo(
       rexCall: RexCall,
       inputNodes: mutable.Map[RexNode, Integer]): PythonFunctionInfo = rexCall.getOperator match {
-    case sfc: ScalarSqlFunction if sfc.getScalarFunction.getLanguage == FunctionLanguage.PYTHON =>
+    case sfc: ScalarSqlFunction =>
```

Comment:
Rename these rexCall to pythonRexCall? both for `createPythonScalarFunctionInfo` and `extractPythonScalarFunctionInfos`.